pkg/plugin/connector/builtin/v1/internal/fromplugin/specifier.go

```diff
@@ -74,7 +74,8 @@ func SpecifierParameter(in cpluginv1.SpecifierParameter) connectorPlugin.Paramet
 	// in.Required is converted to a validation of type ValidationTypeRequired
 	// making sure not to duplicate the required validation
 	if in.Required && !requiredExists {
-		//nolint:makezero // we don't know upfront if we need add this validation
+		//nolint:makezero // regarding ' append to slice `validations` with non-zero initialized length'
```

Comment:
super nitpicky, but it just looked weird 
```suggestion
		//nolint:makezero // regarding 'append to slice `validations` with non-zero initialized length'
```